ui/desktop/app/routes/scopes/scope/projects/targets.js

```diff
@@ -47,10 +47,17 @@ export default class ScopesScopeProjectsTargetsRoute extends Route {
   @action
   async connect(model) {
     try {
-      await this.ipc.invoke('connect', {
+      await this.ipc.invoke('cli');
+      const connect = this.ipc.invoke('connect', {
         target_id: model.target.id,
         token: this.session.data.authenticated.token,
       });
+
+      connect.then((data) => {
+        this.notify.success(JSON.stringify(data));
+      }, (e) => {
```

Comment:
The error case is the same as the next one.  You don't need to handle it here too.